.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: false
 language: php
 php:
   - 5.6
-  - 7
-  - hhvm
+  - 7.0
+  - 7.1
```

Comment:
May be add `nightly` also ;) . https://github.com/auraphp/Aura.Di/blob/2c63b9d0c9d2dbb1e0e11084cb86245ade7266ff/.travis.yml#L8